index-contrast.html

```diff
@@ -5,6 +5,7 @@
     <script
       async
       src="https://www.googletagmanager.com/gtag/js?id=G-9XP6GJSL3K"></script>
```

Comment:
you sure you need both this and the library initialization? quite sure that's not true

Reply:
the react-ga package, apparently doesnt work